lib/http-context.js

```diff
@@ -433,19 +433,28 @@ function toJSON(input) {
   }
 }
 
-function toXML(input) {
+function toXML(input, method) {
   var xml;
+  var returns = method.returns[0];
+  var wrapperElement;
+  if (returns.xml && returns.xml.wrapperElement) {
+    wrapperElement = returns.xml.wrapperElement;
+  } else {
+    wrapperElement = 'response';
+  }
   if (input && typeof input.toXML === 'function') {
     xml = input.toXML();
   } else {
-    if (input) {
+    if (typeof(input) == 'string') {
+      input = { result: input };
+    } else if (typeof(input) == 'object') {
```

Comment:
@bajtos when `input` is string, `js2xmlparser` gets syntax error, then try catch block from `sendBodyXml()` throws 500 , should we modify the data so it doesn't internal server error? or should we not let user pass in just one string.